client/stylesheets/puzzle.scss

```diff
@@ -131,9 +131,15 @@ table.puzzle-list {
   margin: 2px;
 }
 
+@font-face {
+  font-family: "Emoji";
+  src: "Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji";
+  unicode-range: U+1F300-1FAFF, U+1F100-1F1FF, U+2300-23FF, U+2600-27BF;
+}
+
 .answer, #jr-puzzle-guess {
   text-transform: uppercase;
-  font-family: "Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji", monospace;
+  font-family: "Emoji", monospace, "Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji", sans-serif;
```

Comment:
Just out of curiosity: do we expect the additional fallbacks after monospace ("Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji", sans-serif) to actually ever be reached?  (Seems harmless to have them just in case, and if there's a range that isn't covered by the system monospace font one of those others is more likely to have at least *some* glyph, I guess?)

Reply:
Mostly it just felt like good practice. I didn’t prioritize most code points that are basically stylized characters. Ideally, monospace will handle those but in practice I imagine it frequently won’t. Similarly, I expect some unusual scripts (One day, there will be an answer in Linear B...) will fall all the way through and specifying sans feels better than not.

But in practice, I doubt it’s that different from just letting the system fallbacks take over. And if any of those cases actually comes up, we’ll probably patch the font list to handle it.